gtests/net/mptcp_unsupported/mptcp_ip_tos_client.pkt

```diff
@@ -0,0 +1,17 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
```

Comment:
probably it's not strictly needed on the 3WHS, but it should be safer to add --tolerance_usecs=<big value> (so that the test doesn't fail on slow machines)

Reply:
oh wait. scratch this comment